quickstarts/angular.mdx

```diff
@@ -183,63 +179,20 @@ export class AppComponent {
     console.log('loaded', { data });
   };
 }
-
 ```
 
-The Angular component is generated as a wrapper around the original React component.  This approach is clever, as it allows Novu's engineers to focus on creating and developing things in the React way.
+The Angular component is generated as a wrapper around the original React component. This approach is clever, as it allows Novu's engineers to focus on creating and developing things in the React way.
 
 Additionally, many other frameworks can still use the created components using the wrapping approach.
 
-We need to add `@types/react` as dev dependency for the angular component to work properly.
+Now head to the `my-app/tsconfig.json` file, we’re going to add `"allowSyntheticDefaultImports": true` to the `compilerOptions` array.
 
-Open your terminal and navigate to the app root directory and type the following:
-
-```
-npm i [@types/react](https://github.com/types/react)
-
-or
-
-yarn add [@types/react](https://github.com/types/react)
-
-```
-
-Now head to the `my-app/tsconfig.json` file.
-
-And we’re going to add `"allowSyntheticDefaultImports": true` to the `compilerOptions` array.
-
-```json
-/* To learn more about this file see: https://angular.io/config/tsconfig. */
+```diff
 {
-  "compileOnSave": false,
   "compilerOptions": {
-    "allowSyntheticDefaultImports": true,
-    "baseUrl": "./",
-    "outDir": "./dist/out-tsc",
-    "forceConsistentCasingInFileNames": true,
-    "strict": true,
-    "noImplicitOverride": true,
-    "noPropertyAccessFromIndexSignature": true,
-    "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true,
-    "sourceMap": true,
-    "declaration": false,
-    "downlevelIteration": true,
-    "experimentalDecorators": true,
-    "moduleResolution": "node",
-    "importHelpers": true,
-    "target": "ES2022",
-    "module": "ES2022",
-    "useDefineForClassFields": false,
-    "lib": ["ES2022", "dom"]
-  },
-  "angularCompilerOptions": {
-    "enableI18nLegacyMessageIdFormat": false,
-    "strictInjectionParameters": true,
-    "strictInputAccessModifiers": true,
-    "strictTemplates": true
++   "allowSyntheticDefaultImports": true,
```

Comment:
Simplifying this down into a diff, as some users may have a different looking `tsconfig` they are integrating with.